pkg/serverless/aws/logs.go

```diff
@@ -177,3 +179,24 @@ func createStringRecordForReportLog(l *LogMessage) string {
 
 	return stringRecord
 }
+
+// ParseLogsAPIPayload transforms the payload received from the Logs API to an array of LogMessage
+func ParseLogsAPIPayload(data []byte) ([]LogMessage, error) {
+	var messages []LogMessage
+	if err := json.Unmarshal(data, &messages); err != nil {
+		// Temporary fix to handle malformed JSON tracing object : retry with sanitization
+		log.Debug("Can't read log message, retry with sanitization")
+		sanitizedData := removeInvalidTracingItem(data)
+		if err := json.Unmarshal(sanitizedData, &messages); err != nil {
+			return nil, errors.New("can't read log message")
+		}
+		return messages, nil
+	}
+	return messages, nil
+}
+
+// Temporary fix to handle malformed JSON tracing object
```

Comment:
Nit: In Go, the comments above functions are supposed to always begin with the function name -- it's a documentation convention

So, `removeInvalidTracingItem is a temporary fix to...`

Reply:
✅